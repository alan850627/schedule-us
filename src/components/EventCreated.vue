<template>
  <b-container class="new-event">
    <b-row align-h="center">
      <b-col cols="8">
        <b-card class="text-center">
          <h1>Event Created!</h1>
          <p>Copy this link to share with your friends:
          <br><a :href="link">{{ baseURL }}{{ link }}</a></p>
       </b-card>
      </b-col>
    </b-row>
  </b-container>
</template>

<script>
export default {
  name: 'EventCreated',

  components: {
  },

  props: {
  },

  computed: {
    baseURL: function () {
      return `${window.location.host}`
    },
    link: function () {
      return `/#/event/${this.eventId}`
    }
  },

  data () {
    return {
      eventId: ''
    }
  },

  methods: {
  },

  watch: {
  },
}
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped>

</style>
