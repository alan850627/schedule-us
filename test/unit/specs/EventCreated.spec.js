import EventCreated from '@/components/EventCreated'
import { shallow } from 'vue-test-utils'

describe('EventCreated.vue', () => {
  it('has the expected html structure', () => {
    let cmp = shallow(EventCreated, {
    })
    cmp.vm.eventId = 'mL5TyA3eVWObYeUzh5dS'
    expect(cmp.element).toMatchSnapshot()
    expect(cmp.vm).toBeDefined()
  })
})
