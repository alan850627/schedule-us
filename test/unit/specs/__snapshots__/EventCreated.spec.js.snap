// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventCreated.vue has the expected html structure 1`] = `
<b-container
  class="new-event"
>
  <b-row
    align-h="center"
  >
    <b-col
      cols="8"
    >
      <b-card
        class="text-center"
      >
        <h1>
          Event Created!
        </h1>
         
        <p>
          Copy this link to share with your friends:
        
          <br />
          <a
            href="/#/event/"
          >
            /#/event/
          </a>
        </p>
      </b-card>
    </b-col>
  </b-row>
</b-container>
`;
